spec/lib/unwrappr/researchers/github_comparison_spec.rb

```diff
@@ -37,43 +37,74 @@ module Researchers
         context 'given a Github repo' do
           let(:github_repo) { 'envato/unwrappr' }
 
-          context 'given the repo has "vx.x.x" tags' do
-            before do
-              allow(client).to receive(:compare)
-                .with('envato/unwrappr', 'v1.0.0', 'v1.1.0')
-                .and_return(response)
-              allow(client).to receive(:compare)
-                .with('envato/unwrappr', '1.0.0', '1.1.0')
-                .and_return(nil)
-            end
+          context 'given the gem is added' do
+            let(:base_version) { nil }
+            let(:head_version) { GemVersion.new('1.1.0') }
 
-            it 'returns the data from Github' do
-              expect(research).to include(github_comparison: response)
+            it "doesn't add data from Github" do
+              expect(research).to_not include(:github_comparison)
```

Comment:
Took me a minute to parse this assertion. You're just saying the research hast return object does not include the key right?
Is this the rspeccy idiomatic way of doing this? 
I would have done
`expect(research.has_key?(:github_comparison)).to eq(false)` - only to be a little clearer.. Not an issue more of a comment :)

Reply:
How about:
```ruby
expect(research[:github_comparison]).to be_nil
```

Reply:
Added in `46b2319`.